packages/ui/src/pages/Home/MultisigActionMenu.tsx

```diff
@@ -45,30 +49,48 @@ const MultisigActionMenu = ({ withSendButton = true }: MultisigActionMenuProps)
         onClick: () => setIsChangeMultiModalOpen(true)
       })
 
+    // If we are NOT rendering "new transaction button", the functionality will appear in the list
+    if (!withNewTransactionButton) {
+      opts.unshift({
+        text: 'Send transaction',
+        icon: <HiOutlinePaperAirplane size={20} />,
+        onClick: () => setIsSendModalOpen(true)
+      })
+    }
+
     return opts
   }, [
-    getSubscanAccountLink,
+    withNewTransactionButton,
     selectedHasProxy,
     selectedIsWatched,
+    getSubscanAccountLink,
     selectedMultiProxy,
     setIsChangeMultiModalOpen,
-    setIsEditModalOpen
+    setIsEditModalOpen,
+    setIsSendModalOpen
   ])
 
   return (
     <>
-      {withSendButton && (
+      {withNewTransactionButton && (
         <Button
           variant="primary"
           aria-label="send"
           onClick={() => setIsSendModalOpen(true)}
         >
-          New Transaction
+          New transaction
         </Button>
       )}
-      <OptionsMenu options={options} />
+      <OptionsMenu
+        menuButtonBorder={menuButtonBorder}
+        options={options}
+      />
     </>
   )
 }
 
+MultisigActionMenu.defaultProps = {
+  withNewTransactionButton: true
+}
```

Comment:
We've never used this in the past. Please keep the `{ withSendButton = true }` in the props, which shows right away what the default is, and we can remove this.
```suggestion
```